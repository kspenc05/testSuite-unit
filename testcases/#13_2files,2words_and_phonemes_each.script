#!/bin/sh
./cmulookup cmudict-0.7b input/2words.txt input/2words_again.txt