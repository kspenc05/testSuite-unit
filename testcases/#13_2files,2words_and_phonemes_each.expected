G R IY1 N
G L UW1
=====
B AE1 G
B EY1 G AH0 L
=====
DONE! (4 words)
